Automattic/pocket-casts-ios
# Referrals: Update guest pass animation

| 📘 Part of: #1981  |  
|:---:|

Fixes # 

https://github.com/user-attachments/assets/dba2d1e7-237b-426d-abe7-94b3c4ac2e7a

## To test

1. Start the app on a real device
2. Ensure that you have the referrals FF enabled and using an account with Plus or Patron
3. Open the Profile tab
4. Tap on the gift icon on the top left
5. Check the guest-pass radial blur circles move with the position/motion of the device

## Checklist
